draft-ietf-rtcweb-jsep.xml

```diff
@@ -3120,7 +3132,12 @@ candidate:1 1 UDP 1694498815 192.0.2.33 10000 typ host
         setLocalDescription, the following steps MUST be performed:
         <list style="symbols">
 
-          <t>First, the type of the SessionDescription is checked
+          <t>If the description is of type "rollback", follow the
+          processing defined in <xref target="sec.rollback" /> and
+          skip the processing described in the rest of this
+          section.</t>
+
+          <t>The type of the SessionDescription is checked
```

Comment:
I think we probably also need to document in this section the PC states in which a setLD(rollback) is valid. It's not totally clear from the rollback section.